app/src/main/java/com/duckduckgo/app/settings/SettingsViewModel.kt

```diff
@@ -86,6 +87,7 @@ class SettingsViewModel @Inject constructor(
         viewState.value = currentViewState().copy(
             loading = false,
             lightThemeEnabled = isLightTheme,
+                darkModeEnabled = settingsDataStore.darkMode,
```

Comment:
check the indentation

Reply:
Thanks, forgot to apply the styleguide before opening the PR